manifest.json

```diff
@@ -3082,6 +3082,33 @@
                 }
             }
         },
+        "io.github.frozenreflex.neoslinuxexportfix": {
+            "name": "NeosLinuxExportFix",
+            "description": "Fix not being able to export files on Linux Native",
+            "category": "Bug Workarounds",
+            "sourceLocation": "https://github.com/Frozenreflex/NeosLinuxExportFix",
+            "authors": {
+                "Fro Zen": {
+                    "url": "https://github.com/Frozenreflex"
+                }
+            },
+            "flags": [
+                "broken:linux-wine",
+                "broken:windows"
+            ],
```

Comment:
these need to be version flags [docs/manifest-flags](https://www.neosmodloader.com/manifest-flags)

Reply:
https://discord.com/channels/901126079857692714/1011888035924803634/1067251932596555878

Reply:
alright then, il create some prs to update our docs/schema definition